calcom/cal.com
# Outlook - list all calendars

## What does this PR do?



This PR aims to handle paginated MS Graph calls to list all calendars. Currently the limit is 10 calendars per call so we need to fetch the next 10.

Fixes #6007 



**Environment**: Staging(main branch) / Production

## Type of change





## How should this be tested?



- On Outlook create 10 calendars
- Install Outlook to your Cal instance
- On the install calendar page, all calendars should be listed

## Checklist



- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes
